src/modules/typography.scss

```diff
@@ -0,0 +1,183 @@
+  --font-size-7: 28px;
+  --font-size-8: 32px;
+  --font-size-9: 36px;
+  --font-size-10: 42px;
+  --font-size-11: 48px;
+  --font-size-12: 54px;
+  --font-size-13: 60px;
+  --font-size-14: 68px;
+  --font-size-15: 76px;
+  --font-size-16: 84px;
+  --font-size-17: 92px;
+
+  // backward compatibility
+  --dashboard-font: var(--font-family-sans);
+}
+
+// Typography classes
+.heading-1,
+.heading-1-sb,
+.heading-1-lt {
+  font-size: var(--font-size-8);
+  font-weight: var(--font-rg);
+  line-height: 1.25;
+  letter-spacing: 0;
+}
+
+.heading-2,
+.heading-2-sb,
+.heading-2-lt {
+  font-size: var(--font-size-7);
+  font-weight: var(--font-rg);
+  line-height: 1.28;
+  letter-spacing: 0;
+}
+
+@media (max-width: 1152px - 1) {
+  .heading-1,
+  .heading-1-sb,
+  .heading-1-lt {
+    font-size: var(--font-size-7);
+    line-height: 1.28;
+  }
+
+  .heading-2,
+  .heading-2-sb,
+  .heading-2-lt {
+    font-size: var(--font-size-6);
+    line-height: 1.33;
+  }
+}
+
+.heading-3,
+.heading-3-sb,
+.heading-3-lt {
+  font-size: var(--font-size-6);
+  font-weight: var(--font-rg);
+  line-height: 1.33;
+  letter-spacing: 0;
+}
+
+.heading-4,
+.heading-4-sb,
+.heading-4-lt {
+  font-size: var(--font-size-5);
+  font-weight: var(--font-rg);
+  line-height: 1.4;
+  letter-spacing: 0;
+}
+
+.body-1,
+.body-1-sb {
+  font-size: var(--font-size-5);
+  font-weight: var(--font-rg);
+  line-height: 1.4;
+  letter-spacing: 0;
+}
+
+.body-2,
+.body-2-sb {
+  font-size: var(--font-size-4);
+  font-weight: var(--font-rg);
+  line-height: 1.5;
+  letter-spacing: 0;
+}
+
+.body-3,
+.body-3-sb {
+  font-size: var(--font-size-3);
+  font-weight: var(--font-rg);
+  line-height: 1.5;
+  letter-spacing: 0;
+}
+
+.body-4,
+.body-4-sb {
+  font-size: var(--font-size-2);
+  font-weight: var(--font-rg);
+  line-height: 1.4;
+  letter-spacing: 0.16px;
+}
+
+.body-5,
+.body-5-sb {
+  font-size: var(--font-size-1);
+  font-weight: var(--font-rg);
+  line-height: 1.3;
+  letter-spacing: 0.32px;
+}
+
+.interactive-1,
+.interactive-1-sb {
+  font-size: var(--font-size-3);
+  font-weight: var(--font-rg);
+  line-height: 1.5;
+  letter-spacing: 0;
+}
+
+.interactive-2,
+.interactive-2-sb {
+  font-size: var(--font-size-2);
+  font-weight: var(--font-rg);
+  line-height: 1.4;
+  letter-spacing: 0.16px;
+}
+
+.interactive-3,
+.interactive-3-sb {
+  font-size: var(--font-size-1);
+  font-weight: var(--font-rg);
+  line-height: 1.3;
+  letter-spacing: 0.32px;
+}
+
+.helper,
+.helper-sb {
+  font-size: var(--font-size-1);
+  font-weight: var(--font-rg);
+  font-style: italic;
+  line-height: 1.3;
+  letter-spacing: 0.32px;
+}
+
+// font-weight variants
+.heading-1-lt,
+.heading-2-lt,
+.heading-3-lt,
+.heading-4-lt {
+  font-weight: var(--font-lt);
+}
+
+.heading-1-sb,
+.heading-2-sb,
+.heading-3-sb,
+.heading-4-sb,
+.body-1-sb,
+.body-2-sb,
+.body-3-sb,
+.body-4-sb,
+.body-5-sb,
+.interactive-1-sb,
+.interactive-2-sb,
+.interactive-3-sb,
+.helper-sb {
+  font-weight: var(--font-sb);
+}
```

Comment:
Those tokens should be co-located with the typography component already available on albe?